core/src/main/scala/codesearch/core/index/LanguageIndex.scala

```diff
@@ -153,3 +167,13 @@ trait LanguageIndex[VTable <: DefaultTable] { self: DefaultDB[VTable] =>
   private lazy val defaultExtractor: (String, String) => Unit =
     (src: String, dst: String) => Seq("tar", "-xvf", src, "-C", dst) !!
 }
+
+object LanguageIndex {
```

Comment:
docstrings please

Reply:
for what?

Reply:
For everything :trollface: 

Objects and object fields